src/modules/organizations/containers/organization-container.jsx

```diff
@@ -22,13 +23,21 @@ class OrganizationContainer extends React.Component {
     this.props.dispatch(setCurrentOrganization(null));
   }
 
+  updateOrganization(organizationFields) {
+    organizationFields.name = organizationFields.display_name; // eslint-disable-line no-param-reassign
+    this.props.organization.update(organizationFields);
+    this.props.organization.save.then((result) => {
```

Comment:
save isn't being called here. It can also be chained onto update above, `this.props.organization.update(fields).save().then()`